massgov/mayflower
# Patternlab/dp 7916  alert underlines

## Description
The header alert CSS was updated to remove the link styling and border bottoms from the parent element, so that the proper link styling could be applied on the decorative link within the element. In addition to removal of the link styling on the container, there was a minor increase in the line-height to support the extra space needed for the bottom borders on the multiple lines, hence the updated Backstop images.

Note: This PR does include a full-file change, since line endings were changed, but here is a diff of the actual changes:
```
--- /a.css	2018-09-28 12:42:09.000000000 -0400
+++ /b.css	2018-09-28 12:42:19.000000000 -0400
@@ -39,9 +39,8 @@
   }

   &__message {
-    @include ma-link-underline;
     font-size: 1.5rem;
-    line-height: 1em;
+    line-height: 1.3em;

     a {
       display: inline;
@@ -64,19 +63,11 @@

   &__message {

-    &:hover {
-      border-bottom-color: rgba($c-font-base, .5);
-    }
-
     a {
       color: $c-font-base;

       svg {
-        fill: rgba($c-font-base, .5);
-      }
-
-      &:hover {
-        border-bottom-color: transparent;
+        fill: rgba($c-font-base, .5);
       }
     }
   }
```

## Related Issue / Ticket

- [JIRA issue](DP-7916)
- [Github issue]()

## Steps to Test

1. Run local mayflower on this PR's branch:
http://localhost:3000/?p=organisms-header-alert
http://localhost:3000/?p=organisms-header-alert-as-notice

2. On hover, the links should have proper underline styling. (see screenshot below)


## Screenshots
![alert_links](https://user-images.githubusercontent.com/23559449/46221389-f1e9d680-c31a-11e8-87d7-c151b077300f.gif)


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* Any alerts could be impacted by this, although it should only be affecting Header Alert and Header Alert as Notice

#### Today I learned...
